prov/efa/configure.m4

```diff
@@ -134,6 +134,18 @@ AC_DEFUN([FI_EFA_CONFIGURE],[
 			[efadv_support_extended_cq=0])
 	      ])
 
+	AS_IF([test $efadv_support_extended_cq -eq 1],
+		[AC_EGREP_HEADER(
+			[int efadv_wc_read_sgid],
+			[infiniband/efadv.h],
+			[],
+			[
+				efadv_support_extended_cq=0
+				AC_MSG_RESULT([efadv_wc_read_sgid is not available from rdma-core. Do not recover peer address from extended CQ.])
```

Comment:
`Do not recover peer address from extended CQ` is a bit confusing. Maybe just remove it.